checkov/terraform/graph_builder/variable_rendering/renderer.py

```diff
@@ -313,8 +313,8 @@ def _render_dynamic_blocks(self) -> None:
                     try:
                         rendered_blocks = self._process_dynamic_blocks(dynamic_blocks)
                     except Exception:
-                        logging.info(f'Failed to process dynamic blocks in file {vertex.path} of resource {vertex.name}'
-                                     f' for blocks: {dynamic_blocks}', exc_info=True)
+                        logging.warning(f'Failed to process dynamic blocks in file {vertex.path} of resource {vertex.name}'
```

Comment:
I have no problem to remove the `exc_info` but changing to warning will make it visible for normal runs, which we shouldn't.